packages/wallet-ts/src/utils/wallets/metamask/Metamask.ts

```diff
@@ -17,3 +19,22 @@ export const getEthersProviderFromMetamask = async () => {
     throw new WalletException(new Error('Please install Metamask Extension'))
   }
 }
+
+export const updateMetamaskNetwork = async (chainId: EthereumChainId) => {
+  try {
+    const provider = (await detectEthereumProvider({
+      mustBeMetaMask: true,
+    })) as Eip1993ProviderWithMetamask
+
+    if (!provider) {
+      throw new WalletException(new Error('Please install Metamask Extension'))
+    }
+
+    await provider.request({
+      method: 'wallet_switchEthereumChain',
+      params: [{ chainId: `0x${chainId}` }],
```

Comment:
This is not always true tho, it will work for us fine because we are always using `1` and `5` as chainIds, but let's say we want to switch to Kovan (or Arbitrum), Kovan is `42` and in hex its `2A`

Reply:
Also, we should have a `return await` here